lib/transition/import/revert.rb

```diff
@@ -0,0 +1,54 @@
+require 'transition/import/console_job_wrapper'
+
+module Transition
+  module Import
+    module Revert
+      ##
+      # Reverts the import of sites, if they have no mappings or hits - ie no
+      # associated data beyond that which was created by importing the site
+      # YAML file
+      class Sites
+        include Transition::Import::ConsoleJobWrapper
+
+        def initialize(site_abbrs)
+          @site_abbrs = site_abbrs.reject(&:empty?)
```

Comment:
The `reject(&:empty?)` seems to me like it is about cleaning up dodgy input - maybe it belongs in the Rake task?